test/object-store/CMakeLists.txt

```diff
@@ -25,16 +24,13 @@ set(SOURCES
     results.cpp
     schema.cpp
     sectioned_results.cpp
-    set.cpp
+    test_runner.cpp
     thread_safe_reference.cpp
     transaction_log_parsing.cpp
     uuid.cpp
     c_api/c_api.cpp
-    c_api/c_api.c
+    c_api/c_api_file_tests.c
```

Comment:
linking with cmake's OBJECT library exposed a bug that did not support two files of the same name because the system is looking for c_api.o and getting confused because it wasn't there (cmake generates two c_api_<id>.o files to differentiate the two)